about2.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="scss/styles.css" />
+    <link rel="stylesheet" href="scss/about2.css" />
```

Comment:
why do you have two stylesheets in one page?

Reply:
Am re-using styles from the first stylesheet